.github/workflows/build-draft-docs.yml

```diff
@@ -1,38 +1,38 @@
 name: Build Draft Guidelines
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 jobs:
   build_docs:
     strategy:
       matrix:
         document:
-          - 'BR.md'
-          - 'EVG.md'
-          - 'NSR.md'
+          - 'BR'
+          - 'EVG'
+          - 'NSR'
     name: Build ${{ matrix.document }}
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout the code
         uses: actions/checkout@v2
-      - name: Checkout old version for redline (pull request)
+      - name: Checkout old version for redline
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'push' }}
```

Comment:
This is so that we don't produce redlines for manually triggered events, since we have no "thing to redline against" (yes, we could do the previous commit here, but that ended up being messy to get so I didn't bother)

If the `diff_file` doesn't exist (line 26), then the job will still continue, and just skip the redline production.